.github/workflows/deploy.yml

```diff
@@ -20,7 +22,7 @@ jobs:
           fi
       - uses: actions/checkout@v1
       - name: Validate branch name
-        run: ./branchNameValidation.sh $STAGE_PREFIX$branch_name
+        run: ./.github/branchNameValidation.sh $STAGE_PREFIX$branch_name
```

Comment:
This reflects the change in location of our branchNameValidation script.  No logic in it was changed, it was just moved to a more appropriate location.